populateDb/index.js

```diff
@@ -0,0 +1,36 @@
+const { DynamoDBClient, BatchWriteItemCommand } = require( "@aws-sdk/client-dynamodb");
+const ProductsSource = require("../productService/products.json");
+
+(async () => {
```

Comment:
Could you please rewrite it as function that exported from this module?

It seems that it would not work second time for warm lambda